authentication/models.py

```diff
@@ -114,3 +114,8 @@ def get_rank(self, override: int = None) -> str:  # noqa: C901
             return "Pink"
         else:
             return "Initiate"
+
+    @property
+    def ranked_up(self) -> bool:
+        """Determine whether the user has just ranked up."""
+        return self.get_rank(self.gamma) != self.get_rank(override=self.gamma - 1)
```

Comment:
What does the `override` do and why is it only used for one of the calls?

Reply:
Override just forces it to calculate with a different value -- the first call shouldn't have `self.gamma` in it, so that's fixed now.